Text/TextQualityWatchdog/README.md

```diff
@@ -0,0 +1,107 @@
+---
+page_type: sample
+languages:
+- python
+products:
+- azure
+- azure-search
+- azure-cognitive-search
+name: Text Quality Watchdog Custom Skill for Cognitive Search
+description: This custom skill calls a pretrained language model to determine the quality of text extracted from the document cracking process.
+---
+
+# TextQualityWatchdog
+
+This custom skill calls a pretrained language model to determine the quality of text extracted from the document cracking process.
+Text that is well-formed will return a "text_quality_warning" value of 0.
+Corrupted/garbled text resulting from factors such as low scan quality or OCR errors will be caught, and will return a "text_quality_warning" value of 1.
+When included in an index, this value can be used to filter and examine documents that had low quality text extracted during document cracking.
+
+## Requirements
+
+This function has no additional requirements outside the common requirements described in [the root `README.md` file](../../README.md).
+
+## Optional Model Customization
+
+Please note that the provided "watchdog_model.onnx" classifier was trained on a dataset of 18,000 documents, 50% of which were synthetically generated to correct for class imbalance.
+Additionally, labels were generated using an "out-of-vocabulary" metric, which means that a "text_quality_warning" value of 1 does not necessarily indicate corrupted/garbled text.
```

Comment:
> does not necessarily indicate corrupted/garbled text. [](http://example.com/codeflow?start=125&length=54)

nit: would phrase this "means that the data isn't using comon words, which often correlated with garbled text"

Reply:
Done! Rephrased this section about the OOV score in README.md